app/views/games/_chessboard.html.erb

```diff
@@ -1,11 +1,23 @@
  <div class="chessboard">
    <table>
-
+   
    <% (0..7).each do |row| %>
     <tr>
+    
    <% (0..7).each do |column| %>
     <td data-x-position="<%=column%>" data-y-position="<%=row%>">
+    
+    <% piece = @game.piece_data[row][column] %>
```

Comment:
`row` and `column` need to swap here and on line 14 (so it should be `@game.piece_data[column][row]`)